From b7bb7b28899f80d291e77b7080cf66f242b257fb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 22 Oct 2014 15:05:14 +0200
Subject: [PATCH 28/55] contribs: lua: Target MSVC specifically when building
 for it.

This requires a patch in lua that adds the appropriate target
---
 contrib/src/lua/rules.mak | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/contrib/src/lua/rules.mak b/contrib/src/lua/rules.mak
index 8be4af2..c5dfbee 100644
--- a/contrib/src/lua/rules.mak
+++ b/contrib/src/lua/rules.mak
@@ -23,6 +23,9 @@ endif
 ifdef HAVE_SOLARIS
 LUA_TARGET := solaris
 endif
+ifdef HAVE_VISUALSTUDIO
+LUA_TARGET := msvc
+endif
 
 # Feel free to add autodetection if you need to...
 PKGS += lua
-- 
2.1.1

